examples/features/metadata/server/main.go

```diff
@@ -16,7 +16,8 @@
  *
  */
 
-// Binary server is an example server.
+// Server shows how to read metadata from RPC request headers and set
```

Comment:
same: no need to specifically mention header and trailers. may be see different ways